src/services/users/get-invited-user-detail.test.js

```diff
@@ -0,0 +1,49 @@
+const { expect } = require('chai')
+const { faker } = require('@faker-js/faker')
+const {fixtures} = require('../../../test/utils')
+const getInvitedUserDetail = require('./get-invited-user-detail')
+
+
+describe('Get invited-user information', () => {	
+
+	describe('Failure', async()=>{
+		const	invalidInviteToken = fixtures.generateAccessToken({
+			email : faker.internet.email()
+		})
+		it('should throw an error if the invite is invalid', async () =>
+			expect(getInvitedUserDetail(invalidInviteToken))
+				.to.be.rejectedWith(
+					'Invite is invalid/expired')
+		)
+	})
+
+	describe('Success', async()=>{
+
+		let inviteUserData
+		let inviteToken
+		let invites
+
+		before(async () => {
+			inviteUserData = {
+				email : faker.internet.email()
+			}
+
+			inviteToken = fixtures.generateAccessToken(inviteUserData)
+			invites = await fixtures.insertUserInvite(
+				{email: inviteUserData.email})
+		
+		})
+		it('should get invited user information', async () =>{
+	
+			const invitedUser = await getInvitedUserDetail(inviteToken)
+		
+			return expect(invitedUser).to.include({
+				email : invites.email,
+				userId: undefined
```

Comment:
i think the code should be updated if the userId being returned is undefined

Reply:
also we should be able to generate the invite token from the inserted `invites` and compare, they should be the same i think

Reply:
please in this context when you say updated what do you mean?

Reply:
please i just checked and the accessToken is different when  I generate it for the inserted invites

Reply:
> please in this context when you say updated what do you mean?

the code itself (get-invited-user-detail.js) should not return userId since it doesn't seem to be defined anyway and is not used

Reply:
> please i just checked and the accessToken is different when I generate it for the inserted invites

okay, how is it being generated, can you show how the code snippet looks in the test?

Reply:
sure

Reply:
```
describe('Success', async()=>{

		let inviteUserData
		let inviteToken
		let invite

		before(async () => {
			inviteUserData = {
				email : faker.internet.email()
			}

			inviteToken = fixtures.generateAccessToken(inviteUserData)
			invite = await fixtures.insertUserInvite(
				{email: inviteUserData.email})

			console.log(invite.email)
			

		})
		it.only('should get invited user information', async () =>{
	
			const invitedUser = await getInvitedUserDetail(inviteToken)
			// eslint-disable-next-line max-len
			const accessToken = fixtures.generateAccessToken({email : invite.email})
			console.log(invitedUser)
			console.log(accessToken)
			return expect(invitedUser).to.include({
				email : invite.email,
				accessToken


			})

		})	
	})
})
```